ui/webui/test/helpers/storage.py

```diff
@@ -91,6 +91,18 @@ def wait_no_disks_detected(self):
     def wait_no_disks_detected_not_present(self):
         self.browser.wait_not_present("#no-disks-detected-alert")
 
+    def dbus_scan_devices(self):
+        task = self.machine.execute(f'dbus-send --print-reply --bus="{self._bus_address}" \
+            --dest={STORAGE_SERVICE} \
+            {STORAGE_OBJECT_PATH} \
+            {STORAGE_INTERFACE}.ScanDevicesWithTask')
+        task = task.splitlines()[-1].split()[-1]
+
+        self.machine.execute(f'dbus-send --print-reply --bus="{self._bus_address}" \
+            --dest={STORAGE_SERVICE} \
+            {task} \
+            org.fedoraproject.Anaconda.Task.Start')
```

Comment:
I would recommend to switch new things over to busctl as that's part of systemd so always available. However this works so it's fine :)